controllers/batch_controller.go

```diff
@@ -318,3 +320,32 @@ func batchLabels(name string) map[string]string {
 		"app":              "infinispan-batch-pod",
 	}
 }
+
+func BatchResources(spec *v2.BatchContainerSpec) (*corev1.ResourceRequirements, error) {
```

Comment:
We always ignore the returned `error`, so we can simplify this function to also ignore the error return values from `MemoryResources` and `CpuResources`. This should be safe as the webhook always ensures that the values are in the correct format.